src/ios/CDVCamera.m

```diff
@@ -145,9 +201,38 @@ - (void)takePicture:(CDVInvokedUrlCommand*)command
     [self.commandDelegate runInBackground:^{
         
         CDVPictureOptions* pictureOptions = [CDVPictureOptions createFromTakePictureArguments:command];
+        
+        /*
+         FIXME #1
+         What to do about quality?
```

Comment:
I think if the quality option is specified, we need to modify the image retrieved from the PhotoLibraryr or SavedAlbum & (re)save it with the specified quality, then send a 'reference' back to the newly saved image.
This looks like a potential change in the API and might need to be ran past the dev mailing list.